### PR TITLE
Add helper method to generate quotes

### PR DESCRIPTION
**Pull request by gardncl**
# Why 
There is going to be a long list of quotes to choose from and a method for choosing them so this won't fit in the body of the HTML. The quote generator should be split out separately. 

# What
 - [x] New method to return a string that's a quote
 - [x] Render this quote in the main application

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import './App.css';
+import './randomQuote.ts';
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
-        <p>"To be honest you guy don't smell as bad as I thought you would"</p>
+        <randomQuote />
```
Comment (gardncl) [inline src/App.js]: have also tried with this wrapped in p tag

Comment (amast09) [reply src/App.js]: `<p>{randomQuote()}</p>`

Comment (gardncl) [reply src/App.js]: This does not work. `'randomQuote' is not defined no-undef`

Comment (gardncl) [reply src/App.js]: Because I needed this import:
```javascript
import randomQuote from './randomQuote.ts';
```

**Pull request merged**